code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -608,6 +608,723 @@
+			var/datum/status_effect/chem/enthrall/E = L.has_status_effect(/datum/status_effect/chem/enthrall)
+			switch(E.phase)
+				if(3 to INFINITY)//Tier 3 only
+					L.Stun(40 * power_multiplier)
+					E.cooldown += 8
+					addtimer(CALLBACK(GLOBAL_PROC, .proc/to_chat, L, "<span class='notice'>Your muscles freeze up!</b></span>"), 5)
+
+	//HALLUCINATE
+	else if(findtext(message, hallucinate_words))
+		for(var/V in listeners)
+			var/mob/living/carbon/C = V
+			var/datum/status_effect/chem/enthrall/E = C.has_status_effect(/datum/status_effect/chem/enthrall)
+			switch(E.phase)
+				if(3 to INFINITY)//Tier 3 only
+					new /datum/hallucination/delusion(C, TRUE, null,150 * power_multiplier,0)
+
+	//HOT
+	else if(findtext(message, hot_words))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			var/datum/status_effect/chem/enthrall/E = L.has_status_effect(/datum/status_effect/chem/enthrall)
+			switch(E.phase)
+				if(3 to INFINITY)//Tier 3 only
+					L.adjust_bodytemperature(50 * power_multiplier)//This seems nuts, reduced it, but then it didn't do anything, so I reverted it.
+					addtimer(CALLBACK(GLOBAL_PROC, .proc/to_chat, L, "<span class='notice'>You feel your metabolism speed up!</b></span>"), 5)
+
+	//COLD
+	else if(findtext(message, cold_words))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			var/datum/status_effect/chem/enthrall/E = L.has_status_effect(/datum/status_effect/chem/enthrall)
+			switch(E.phase)
+				if(3 to INFINITY)//Tier 3 only
+					L.adjust_bodytemperature(-50 * power_multiplier)//This
+					addtimer(CALLBACK(GLOBAL_PROC, .proc/to_chat, L, "<span class='notice'>You feel your metabolism slow down!</b></span>"), 5)
+
+
+	//GET UP
+	else if(findtext(message, getup_words))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			var/datum/status_effect/chem/enthrall/E = L.has_status_effect(/datum/status_effect/chem/enthrall)
+			switch(E.phase)
+				if(3 to INFINITY)//Tier 3 only
+					if(L.resting)
+						L.lay_down() //aka get up
+					L.SetStun(0)
+					L.SetKnockdown(0)
+					L.SetUnconscious(0) //i said get up i don't care if you're being tased
+					E.cooldown += 10 //This could be really strong
+					addtimer(CALLBACK(GLOBAL_PROC, .proc/to_chat, L, "<span class='notice'>You jump to your feel from sheer willpower!</b></span>"), 5)
+
+	//PACIFY
+	else if(findtext(message, pacify_words))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			var/datum/status_effect/chem/enthrall/E = L.has_status_effect(/datum/status_effect/chem/enthrall)
+			switch(E.phase)
+				if(3)//Tier 3 only
+					E.status = "pacify"
+					E.cooldown += 10
+
+	//CHARGE
+	else if(findtext(message, charge_words))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			var/datum/status_effect/chem/enthrall/E = L.has_status_effect(/datum/status_effect/chem/enthrall)
+			switch(E.phase)
+				if(3)//Tier 3 only
+					E.statusStrength = 2* power_multiplier
+					E.status = "charge"
+					E.cooldown += 10
+
+
+	/* THE MAYBE PILE
```

Comment:
we in general do not want too much commented out code left in without good reason. there's an exception on places like rp server but main is a tgcode codebase so we'd like to stick to tg standards whenever possible.

Reply:
That's fine, I'll remove it, it was more so you guys can ask for any to be implemented if you wanted!